egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
```

Comment:
Since Pawn there is no data really associated with the pawn, I just made the class an object placeholder until we come up with needs for it.